alembic/versions/fe6dcd0178f7_add_cbs_cities_table_to_hold_data_from_.py

```diff
@@ -0,0 +1,55 @@
+"""Add cbs_cities table to hold data from cities.csv
+
+Revision ID: fe6dcd0178f7
+Revises: fd0c85b6d1bf
+Create Date: 2022-04-26 20:41:18.049443
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "fe6dcd0178f7"
+down_revision = "fd0c85b6d1bf"
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pylint: disable=E1101
+def upgrade():
+    op.create_table(
+        "cbs_cities",
+        sa.Column("heb_name", sa.String()),
+        sa.Column("yishuv_symbol", sa.Integer(), primary_key=True),
+        sa.Column("eng_name", sa.String()),
+        sa.Column("district", sa.Integer()),
+        sa.Column("napa", sa.Integer()),
+        # sa.Column('natural_zone', sa.Integer(), nullable=True),
+        sa.Column("municipal_stance", sa.Integer(), nullable=True),
+        sa.Column("metropolitan", sa.Integer(), nullable=True),
+        # sa.Column('religion', sa.Integer(), nullable=True),
+        # sa.Column('population', sa.Integer(), nullable=True),
+        # sa.Column('other', sa.Float(), nullable=True),
+        # sa.Column('jews', sa.Float(), nullable=True),
+        # sa.Column('arab', sa.Float(), nullable=True),
+        # sa.Column('founded', sa.Integer(), nullable=True),
+        # sa.Column('tzura', sa.Integer(), nullable=True),
+        # sa.Column('irgun', sa.Integer(), nullable=True),
```

Comment:
Why the comment out code?

Reply:
Only a part of the fields in `cities.csv` are loaded, so I left the unloaded fields as comments to make it easier to add a field in case it will be needed.